.github/workflows/build.yml

```diff
@@ -43,6 +50,9 @@ jobs:
       run: |
         cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Release
         cmake --build ${{github.workspace}}/build
+      env:
```

Comment:
This change is in here simply because we didn't update the build.yml file when we were adding a quick stop gap solution for restoring older Linux distros.